app/graphql/resolvers/wallet_resolver.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module Resolvers
+  class WalletResolver < Resolvers::BaseResolver
+    include AuthenticableApiUser
+    include RequiredOrganization
+
+    description 'Query a single wallet'
+
+    argument :id, ID, required: true, description: 'Uniq ID of the wallet'
+
+    type Types::Wallets::SingleObject, null: true
+
+    def resolve(id: nil)
```

Comment:
is this necessary if id is required?

Reply:
id is required but it's a ruby formatting, since it's a named params, it's better to initialize it to `nil`!